src/panel_state_manager.cpp

```diff
@@ -1096,6 +1098,19 @@ void PanelStateManager::setCEState()
     }
 }
 
+bool PanelStateManager::isFunctionSupported(const types::FunctionNumber funcNum)
+{
+    std::vector<types::FunctionNumber> supportedFuncs = {21, 22, 34, 65, 66,
```

Comment:
We have a type defined for this "FunctionalityList". Please use that for better readability.